src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,55 @@
 import React from 'react';
-
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
+
+export const MovieCard: React.FC<Movie> = (props) => {
```

Comment:
You should pass a movie object here and add the correct type for the props 
```suggestion
export const MovieCard: React.FC<Props> = ({ movie }) => {
```